### ISSUE TITLE
Ability to delete project phase

### ISSUE THREAD

**Issue by RishiRajSahu** [appirio-tech/connect-app#2094]
If user wants to change the product of a phase, he/she needs to delete the phase and add a new one

**Comment by maxceem** [comment]
@RishiRajSahu can a user delete all phases? Or if a user has only one phase, there should be no Delete button?

**Comment by RishiRajSahu** [comment]
Yes user can delete all the phases. we can show this placeholder https://marvelapp.com/8a94680/screen/43925162 to customer in case of no phases or all phases unplanned.

**Comment by maxceem** [comment]
@RishiRajSahu done via https://github.com/appirio-tech/connect-app/commit/9bef3905fce41f3d4f6fb1ad4fb254bdac2b86a7

**Comment by maxceem** [comment]
fixed `Active and Delivered phases can't be deleted` via https://github.com/appirio-tech/connect-app/commit/35f8ae91160944a2040407bf7b4ae70f0bdb8fcd
